pkg/config/config.go

```diff
@@ -411,26 +411,24 @@ func (lci *LocalConfigInfo) writeToFile() error {
 
 // GetType returns type of component (builder image name) in the config
 func (lc *LocalConfig) GetType() string {
-	if lc.componentSettings.Type == nil {
+	return getStringOrEmpty(lc.componentSettings.Type)
+}
+
+func getStringOrEmpty(ptr *string) string {
```

Comment:
I think we should move this to a utils packages as this can helpful at other places too like preference.go etc too.

Reply:
Done.